<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimaticWidow</class>
 <widget class="QMainWindow" name="AnimaticWidow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnimaticWidow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>动画设置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>动画</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <property name="bottomMargin">
         <number>7</number>
        </property>
        <item row="2" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2" colspan="2">
           <widget class="QComboBox" name="comboBox_fileforplay">
            <item>
             <property name="text">
              <string>所有文件</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>最近5个文件</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>最近10个文件</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>最近15个文件</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>最近20个文件</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>仰角层：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QComboBox" name="comboBox_10"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>循环播放：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QTableWidget" name="tableWidget_filelist">
          <column>
           <property name="text">
            <string>文件名</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButtonSelectDir">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButtonClearList">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAsdfasd">
   <property name="text">
    <string>asdfasd</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
